setup.py

```diff
@@ -18,21 +19,20 @@
     ],
     entry_points={
         "console_scripts": [
-            "pyfxa-versions = ???:main",
+            "pyfxa-versions = pyfxa_versions:main",
         ]
     },
-    include_package_data=True,
```

Comment:
Adding data files is tricky. This line will look for data files using svn and cvs, neither of which we use. If you install setuptools-git (https://pypi.org/project/setuptools-git/), then I think this would work by itself.

However, we're also specifying `package_data` which says what data files to use. I'm pretty sure that's getting ignored because of this line and not having setuptools-git installed.

If I nix this line and fix `package_data`, then the sdist gets built with the data file in it and when installing, it installs the data file.

I'm testing using `make build` which cleans and then creates a source dist. Looking at the output, one can determine what actually made it into the .tar.gz file.

There's a long drawn-out explanation of including data here which I sort of don't understand, but maybe it's helpful:

https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=include_package_data#including-data-files

In my projects, I tend to have a `MANIFEST.in` file and use `check-manifest` (https://pypi.org/project/check-manifest/) to keep it up-to-date. Having said that, now that I've spent an hour looking at packaging data I'm pretty sure I've got it wrong and I should go back and rewrite everything.

Reply:
yeah, sadly, that's the documentation I was reading when I said, "I know, @willkg will know how to handle this crap I don't understand here" :)